terraform_cft/00_projects/variables.tf

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2019 Google LLC. This software is provided as-is, without warranty
+ * or representation for any use or purpose. Your use of it is subject to your 
+ * agreement with Google.  
+ */
+variable "gcp_billing_id" {
+  description = "The billing id to use with projects."
+}
+
+variable "gcp_org_id" {
+  description = "The ID of the GCP organization where the projects will be created"
+}
+
+variable "folder_id" {
+  description = "The ID of the folder where the projects will be created"
+}
+
+variable "shared_services_project" {
```

Comment:
Could we provide defaults?

Reply:
Cannot provide default value to this one since it references the project ID, which will be different for each person deploying this.